guided-trials-default-custom-user-data/default-custom-user-data.gradle

```diff
@@ -135,6 +135,17 @@ void addCiMetadata() {
             buildScan.value 'CI agent ID', agentId
         }
     }
+
+    if (isGitHubActions()) {
+        def repo = System.getenv('GITHUB_REPOSITORY')
+        def runId = System.getenv('GITHUB_RUN_ID')
+        if (repo && runId) {
```

Comment:
is this a precaution or are those two variables always set when using GH actions ?

Reply:
Yes, just a precaution. Do you think it's unnecessary?

Reply:
let's keep it. This is used in customers projects.